r/configure

```diff
@@ -413,7 +413,7 @@ CXX17FLAGS=`"${R_HOME}"/bin/R CMD config CXX17FLAGS`
 CXX17STD=`"${R_HOME}"/bin/R CMD config CXX17STD`
 CPPFLAGS=`"${R_HOME}"/bin/R CMD config CPPFLAGS`
 TEST_CMD="${CXX17} ${CPPFLAGS} ${PKG_CFLAGS} ${CXX17FLAGS} ${CXX17STD} -xc++ -"
-echo "#include $PKG_TEST_HEADER" | ${TEST_CMD} >/dev/null 2>&1
+TEST_ERROR=$(echo "#include $PKG_TEST_HEADER" | ${TEST_CMD} -o /dev/null 2>&1)
```

Comment:
>I also think you have to use backticks instead of $() because of bashisms.

Nope I checked that with `checkbashism` the only thing flagged was the `echo "\n"` ^^

>the previous command 

Yes and the previous command is the `$()` so unless we add something in between running the command and checking the exit code it will be fine (which is also the case for your solution).